src/main/java/pl/allegrotech/productsshop/infrastructure/InMemoryProductRepository.java

```diff
@@ -0,0 +1,34 @@
+package pl.allegrotech.productsshop.infrastructure;
+
+import org.springframework.stereotype.Repository;
+import pl.allegrotech.productsshop.domain.Product;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Repository
+class InMemoryProductRepository implements ProductRepository {
+
+    private final Map<String, Product> products = new HashMap<>();
+
+    @Override
+    public int count() {
```

Comment:
potrzebne to?

Reply:
Nie, z tego co widzę było na potrzeby testów tylko. Fixed.